code/src/index.css

```diff
@@ -1,13 +1,236 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  width: auto;
+  background-color: rgb(22, 22, 22);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* THIS TAKES CARE OF SEPARATION BETWEEN ARTISTS */
+.artist-name:not(:first-child, :last-child)::before {
+  color: #a0a0a0;
+  content: ", ";
+}
+
+.artist-name:last-child::before {
+  content: " & ";
+  color: #a0a0a0;
+}
```

Comment:
Can you explain this more? How does it work?

Reply:
TBH, I'm not entirely sure.  One of the recommended Stack questions/answers for this week dealt with comma separation.  And, we pretty much copied the CSS solution exactly.  It worked so, I didn't do any further research. 

https://stackoverflowteams.com/c/technigo/questions/2239